src/__tests__/posthog-core.js

```diff
@@ -333,3 +334,36 @@ describe('__compress_and_send_json_request', () => {
         expect(given.overrides._send_request.mock.calls).toMatchSnapshot()
     })
 })
+
+describe('utm tags', () => {
```

Comment:
For organization, describe blocks should focus on specific functions (e.g. `capture_pageview()`) rather than behavior.